requirements/pkgutils.txt

```diff
@@ -4,7 +4,7 @@ flake8>=3.8.3
 flakeplus>=1.1
 flake8-docstrings>=1.7.0
 pydocstyle==6.3.0
-tox>=3.8.4
+tox>=3.8.4,<5.9.0
```

Comment:
is it 4.9 or 5.9? I saw that the latest release is 4.11 https://pypi.org/project/tox/

Reply:
but seems tests are passing!

Reply:
> but seems tests are passing!

I was just lazy checking tox 4.11 - that tmp commit is removed and the whole PR is refactored @auvipy 

Let's hope it fixes the issue! 🙏